component/admin/controllers/icalrepeat.php

```diff
@@ -171,7 +171,7 @@ function edit($key = NULL, $urlVar = NULL)
 
 		$db = JFactory::getDbo();
 		$query = "SELECT rpt.eventid"
-				. "\n FROM (#__jevents_vevent as ev, #__jevents_icsfile as icsf)"
+				. "\n FROM #__jevents_vevent as ev, #__jevents_icsfile as icsf"
```

Comment:
I'm presuming this still works in MySQL ?

Should probably be an INNER JOIN

Reply:
Maybe, but I'm just improving mariadb support :P